assets/sass/components/global/_googlesitekit-publisher-wins.scss

```diff
@@ -74,6 +74,13 @@
 		max-width: 700px;
 	}
 
+	&.googlesitekit-publisher-win--full {
+
+		.googlesitekit-publisher-win__desc {
+			max-width: none;
```

Comment:
Looking at the initial screenshot, it seems the more consistent change here would be to apply the max-width on the unbounded text rather than removing it for this one notice.

![](https://user-images.githubusercontent.com/73545194/170661333-38bcbba4-4eb8-48be-80d6-397d01abe079.png)

Unless there is some reason we can't, let's apply the same max with to both areas instead.

Reply:
No problem @aaemnnosttv, I've made that change :+1: